clang/test/CodeGenCXX/cheri/csetbounds-stats-all.cpp

```diff
@@ -42,7 +43,8 @@ void test_onstack_int_overaligned(struct Foo *s) {
 
 void test__stack_array(void) {
   char buf[333];
-  do_stuff_with_buf(buf); // CSV-NEXT: 0,333,?,"{{.+}}/csetbounds-stats-all.cpp:45:21","Add subobject bounds","array decay for char[333]"
+  do_stuff_with_buf(buf); 
```

Comment:
Leave these alone

Reply:
The thing here is that the line number `45` is hard coded, by adding the `// REQUIRES:` cause causes this test to fail. All the other assertions use the `[[@LINE]]` directive to get the line making this clause an outlier. Is there a specific reason why this assertion needs to be this way?